tests/SignalSlot/CreateUserGroupSlotTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\Tests\SignalSlot;
+
+use eZ\Publish\Core\SignalSlot\Signal\UserService\CreateUserGroupSignal;
+
+class CreateUserGroupSlotTest extends AbstractPublishSlotTest
+{
+    public function createSignal()
+    {
+        return new CreateUserGroupSignal([
+            'userGroupId' => $this->contentId
+        ]);
+    }
+
+    public function getReceivedSignalClasses()
+    {
+        return ['eZ\Publish\Core\SignalSlot\Signal\UserService\CreateUserGroupSignal'];
```

Comment:
Could be `CreateUserGroupSignal::class` instead. Also true for some other places in this PR.

Reply:
Done.